utils/nctl/sh/assets/compile.sh

```diff
@@ -6,7 +6,15 @@
 #   NCTL - path to nctl home directory.
 ########################################
 
-source "$NCTL"/sh/assets/compile_node.sh
+pull optional flag to run in debug mode
+while getopts 'd' opt; do 
+    case $opt in
+        d ) export NCTL_COMPILE_TARGET="debug";;
```

Comment:
Handling the parameter like that will result in the following:
```
magister@zacisze:~/Casper/casper-node$ nctl-compile-node
Finished release [optimized] target(s) in 7m 03s
magister@zacisze:~/Casper/casper-node$ nctl-compile-node -d
Finished dev [unoptimized + debuginfo] target(s) in 0.12s
magister@zacisze:~/Casper/casper-node$ nctl-compile-node
Finished dev [unoptimized + debuginfo] target(s) in 0.12s
```

So the third time I started it **without** `-d` flag, yet it still compiled the `dev`.

Reply:
Works as intended, thanks :+1: